### DISCUSSION TITLE
Search.hbubli.cc down for a bit

### DISCUSSION THREAD

**Discussion by HBubli** [searxng/searx-instances#411] [Report instance downtime]
Alright seems like plesk firewall completely screwed up the server, gonna take a bit for the recovery system to boot etc, or in the worst case scenario, the backup to apply. No clue how long its gonna take.

**Comment by unixfox** [comment]
Thanks for the notification.

**Comment by HBubli** [comment]
Well it was a quick fix, just had to delete the iptables rules, its back up again.
